conftest.py

```diff
@@ -46,3 +73,36 @@ def mock_lsf_version(monkeypatch, request):
     except OSError:
         # Provide a fake implementation of lsf_version()
         monkeypatch.setattr(dask_jobqueue.lsf, "lsf_version", lambda: "10")
+
+
+all_envs = {
+    None: LocalCluster,
+    "pbs": PBSCluster,
+    "moab": MoabCluster,
+    "slurm": SLURMCluster,
+    "sge": SGECluster,
+    "lsf": LSFCluster,
+    "oar": OARCluster,
+    "htcondor": HTCondorCluster,
+}
+
+
+@pytest.fixture
+def EnvSpecificCluster3(pytestconfig):
```

Comment:
Not sure about the final `3` here? Not sure about this definition at all... Could you explain?